### ISSUE TITLE
Descend into forms other than load and require

### ISSUE THREAD

**Issue by jschaf** [jschaf/esup#30]
From https://www.reddit.com/r/emacs/comments/3v7c5b/working_on_your_emacs_config_init_esup_is_very/cxlviky

Useful for org-mode configs

**Comment by jschaf** [comment]
esup now uses advice to hook into require and load, so everything should just work.  Fixed by 81f5f4da26b2b9e8c4ce9cc0272c3fb7ea4dbf3e

**Comment by MikeTheGreat** [comment]
I tried **esup** today on a org-mode config file and got the following results:


```
Total User Startup Time: 4.292sec     Total Number of GC Pauses: 29     Total GC Time: 0.362sec

init.el:6  4.292sec   99%
(org-babel-load-file "~/.emacs.d/configuration.org")

init.el:5  0.000sec   0%
(setq org-element-use-cache nil)
```
I just did the `M-x esup` thing - Am I doing something wrong / not doing something I should?
